aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +260,40 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Forms in layers vs. full page 
+
```

Comment:
This is changed

Reply:
@bezAdane and @britt6612 - 

I am going to have to come back to review this. I will provide detailed comments and suggestions when I do.

We are getting closer here, but we are still missing the focus on the user and what is the user trying to accomplish. We need to be speaking to the user problems being solved for and then which tool someone should reach for when solving that problem. How can we tie this content back to representative use cases?

If we look at the [original question](https://www.figma.com/file/Lh01flLH73cFNGZiFxEFth/HPE---Office-Hours?node-id=460%3A6264), it is framed in use cases. "In these types of situations, I think we should use A, because of x, y, z reasons/benefits. In this of other type of situation, we should use B because of m, n, o benefits. We need to be presenting our content in this sort of rational flow --> "In this type of situation where the user is trying to accomplish _____, we recommend using this tool/variant/etc., because it provides these benefits and value to the end user."